installer/tue-install-impl.bash

```diff
@@ -1018,6 +1055,24 @@ do
     case "$1" in
         --debug) DEBUG=true
             ;;
+        --no-ros-deps) NO_ROS_DEPS=true
+            ;;
+        --doc-depend)
+        # shellcheck disable=SC2034
+        TUE_INSTALL_DOC_DEPEND=true
+            ;;
+        --no-doc-depend)
+        # shellcheck disable=SC2034
+        TUE_INSTALL_DOC_DEPEND=false
+            ;;
+        --test-depend)
+        # shellcheck disable=SC2034
+        TUE_INSTALL_TEST_DEPEND=true
+            ;;
+        --no-test-depend)
+        # shellcheck disable=SC2034
+        TUE_INSTALL_TEST_DEPEND=false
+            ;;
```

Comment:
Maybe set the default value of TUE_INSTALL_TEST_DEPEND and TUE_INSTALL_DOC_DEPEND to be false. If the option is used then the value is set as true.

Reply:
The default is what you put in your config

Reply:
That's a bit messy in my opinion. We should keep the defaults here and overrides in the config. As a typical user will overlook this config. The ones who have the knowledge about how tue-env works will use the config file correctly.

Reply:
If you don't have a config, the default is false.

Reply:
If there exists no config file then these options won't work as the variables are not exported.

Reply:
Ok, then we export them.